### PR TITLE
Switch formatDateFromNow() to use moment.humanize()

### PR DESCRIPTION
**Pull request by rszwajko**
image

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by sjd78** [comment]
ci test

File: src/helpers.js
Comment (sjd78) [inline src/helpers.js:257]: Why not just use `formatUptimeDuration` in `src/utils/format.js`?  Then the uptime string and the age of snapshot would all follow the same format on the VM Details page.

Comment (rszwajko) [reply src/helpers.js:257]: My assumption was that we want to keep single number "humanized" format.

Comment (sjd78) [reply src/helpers.js:257]: Historically, there was a bunch of debate over the format of the text for how long ago type of things.  The debate around the uptime one was annoyingly long.

So at this point, as long as `humanize()` has localized text provided by moment, then we're good.  If not, we'll need to drop back to the `moment-duration-format` and our own localization strings.

Comment (sjd78) [reply src/helpers.js:257]: @rszwajko , @sgratch , @hstastna - As long as the humanized string are translated, I don't mind using it instead of a more uptime string format.

File: src/helpers.js
Comment (sjd78) [inline src/helpers.js:254]: A few nit-picks:

  - I suggest moving this function to `src/utils/format.js` to be in the same place as `formatUptimeDuration`.

  - Maybe a slightly different name?  `formatHowLongAgo`

  - `msg` isn't needed since `humanize(true)` handles that for us

  - `s/date/pastDate/` maybe?

**Review by sjd78 — CHANGES_REQUESTED** [review]

**Comment by rszwajko** [comment]
@sjd78 
1. implemented suggested changes
2. (successfully) tested that 'moment' supports all languages required by ovirt

**Review by sjd78 — APPROVED** [review]
LGTM

**Comment by sgratch** [comment]
@rszwajko for some reason both pt_BR and zh_CN locales are not translated by 'moment' for me for snapshots duration. Please approve that both are working as expected for snapshots before merging

**Comment by rszwajko** [comment]
> @rszwajko for some reason both pt_BR and zh_CN locales are not translated by 'moment' for me for snapshots duration. Please approve that both are working as expected for snapshots before merging

@sgratch  @sjd78 
This is regression after #1433. It seems that adding content to file `src/intl/translated-time-durations.json` disabled some translations.

**Comment by sjd78** [comment]
> > @rszwajko for some reason both pt_BR and zh_CN locales are not translated by 'moment' for me for snapshots duration. Please approve that both are working as expected for snapshots before merging
> 
> @sgratch @sjd78
> This is regression after #1433. It seems that adding content to file `src/intl/translated-time-durations.json` disabled some translations.

Really?  This code is what extends the moment locale to add our translations for moment-duration-format which does not ship with any translations:
https://github.com/oVirt/ovirt-web-ui/blob/04f3a7e10ba4fa93946a84b38f61a574850a8eb9/src/intl/initialize.js#L82-L137

Does humanize() use the same keys?  I thought that `moment-duration-format` used keys unique to the plugin.

moment locale customization: https://momentjs.com/docs/#/customization/

**Comment by sgratch** [comment]
@rszwajko @sjd78 

> Really? This code is what extends the moment locale to add our translations for moment-duration-format which does not ship with any translations

It seems that for fixing this we just need to rename translated-time-durations.json locale entries as follows (it's case sensitive):
pt-BR":  => pt-br":
"zh-CN": => "zh-cn":

**Comment by rszwajko** [comment]
> It seems that for fixing this we just need to rename translated-time-durations.json locale entries as follows (it's case sensitive):
> pt-BR": => pt-br":
> "zh-CN": => "zh-cn":

Actually the upper-case version seems correct to me. Let's limit the scope of the changes to moment initialization.
Please check PR #1436

**Review by sgratch — APPROVED** [review]
LGTM

**Comment by isaranova** [comment]
LGTM

**Pull request merged**
